core/lib/object_store/src/mock.rs

```diff
@@ -45,4 +45,8 @@ impl ObjectStore for MockStore {
         bucket_map.remove(key);
         Ok(())
     }
+
+    fn get_storage_prefix_raw(&self, bucket: Bucket) -> String {
+        format!("{bucket}")
```

Comment:
Nit: You can use `bucket.to_string()` here.

Reply:
corrected